QKSMS/src/androidTest/java/com/moez/QKSMS/ui/GettingStartedSkipTest.java

```diff
@@ -0,0 +1,104 @@
+package com.moez.QKSMS.ui;
+
+
+import android.support.test.espresso.ViewInteraction;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.test.suitebuilder.annotation.LargeTest;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+import com.moez.QKSMS.R;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.*;
+import static android.support.test.espresso.assertion.ViewAssertions.*;
+import static android.support.test.espresso.matcher.ViewMatchers.*;
+import static org.hamcrest.Matchers.allOf;
+
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+public class GettingStartedSkipTest {
```

Comment:
This class will need to have an @Before method added that sets the shared prefs correctly to display the welcome screen consistently. "sharedPreferences.putBoolean(SettingsFragment.WELCOME_SEEN, false);". We can add that in another commit/pull request though.